src/utils/errorResponse.js

```diff
@@ -0,0 +1,6 @@
+export default (res, status, message) => {
+  res.status(status).json({
+    success: false,
+    message,
```

Comment:
the message key should be a string of the error message, and also add error which should be an object in case the whole error object needs to be sent to the client side